specifications/api/swagger.yaml

```diff
@@ -344,27 +375,42 @@ x-stakePoolMetrics: &stakePoolMetrics
       description: Number of blocks produced by a given stake pool
 
 x-networkInformationSyncProgress: &networkInformationSyncProgress
-  <<: *percentage
+  <<: *syncProgress
+  description: |
+    Estimated synchronization progress of the node with the underlying network. Note that this may
+    change quite arbitrarily as the node switches to different shorter or longer chain forks.
 
 x-networkInformationBlockchainHeight: &networkInformationBlockchainHeight
   <<: *numberOfSlots
 
-x-networkInformationLocalHeight: &networkInformationLocalHeight
-  <<: *numberOfSlots
-
 x-networkInformationNtpStatus: &networkInformationNtpStatus
   type: object
+  description: |
+    [Network Time Protocol](https://fr.wikipedia.org/wiki/Network_Time_Protocol) information of the server.
```

Comment:
This should probably link to `https://en.wikipedia.org/wiki/Network_Time_Protocol`, unless we really do want to link to the French language version? :)

Reply:
Whoops.